src/Hexonet/Dac.php

```diff
@@ -115,14 +112,18 @@ protected function processResponse(ResponseInterface $response, array $domains):
         }
 
         $lines = explode("\n", $response->getBody()->__toString());
-        $arrayDotData = collect($lines)
+        /** @var \Illuminate\Support\Collection $arrayDotDataCollection */
+        $arrayDotDataCollection = collect($lines);
+
+        $arrayDotData = $arrayDotDataCollection
             ->reduce(function (array $data, string $line) {
                 /** @var string $line DAC result line e.g., PROPERTY[DOMAINCHECK][2]=210 Available */
                 if (!$line = trim($line)) {
                     // skip this line
                     return $data;
                 }
 
+                // ToDo: Evalue if this is the best approach.
```

Comment:
@uphlewis double check this, `$line is string but passed to parse_str 2nd param which is expected to be an array.`

Reply:
I think this is ok - inside the closure it has to be a string due to the strict type